doc/classes/EditorTranslationParserPlugin.xml

```diff
@@ -4,22 +4,39 @@
 		Plugin for adding custom parsers to extract strings that are to be translated from custom files (.csv, .json etc.).
 	</brief_description>
 	<description>
-		Plugins are registered via [method EditorPlugin.add_translation_parser_plugin] method. To define the parsing and string extraction logic, override the [method parse_text] method in script.
+		Plugins are registered via [method EditorPlugin.add_translation_parser_plugin] method. To define the parsing and string extraction logic, override the [method parse_file] method in script.
 		The extracted strings will be written into a POT file selected by user under "POT Generation" in "Localization" tab in "Project Settings" menu.
 		Below shows an example of a custom parser that extracts strings in a CSV file to write into a POT.
 		[codeblock]
 		tool
 		extends EditorTranslationParserPlugin
 
-		func parse_text(text, extracted_strings):
+
+		func parse_file(path, extracted_strings):
+		    var file = File.new()
+		    file.open(path, File.READ)
+		    var text = file.get_as_text()
 		    var split_strs = text.split(",", false, 0)
 		    for s in split_strs:
 		        extracted_strings.append(s)
 		        #print("Extracted string: " + s)
 
+
 		func get_recognized_extensions():
 		    return ["csv"]
 		[/codeblock]
+		[b]Note:[/b] If you override parsing logic for standard script types (gd, C# etc.), it would be better to load the [code]path[/code] argument using [method ResourceLoader.load]. This is because built-in scripts are loaded as [Resource] type, not [File] type.
```

Comment:
```suggestion
		[b]Note:[/b] If you override parsing logic for standard script types (GDScript, C#, etc.), it would be better to load the [code]path[/code] argument using [method ResourceLoader.load]. This is because built-in scripts are loaded as [Resource] type, not [File] type.
```